### ISSUE TITLE
--number  10 only generates 1 model

### ISSUE THREAD

**Issue by graik** [strubelab/multiprot#20]
After fixing #19 (type-casting to int in Ranch.__init__) the run only outputs a single model anyway: 

~/data/py/multiprot/multiprot/scripts/multipr --chain MTG ../domains/frb_1NSG.pdb TGGSGSGSGSGSGSGSGSGSGSTGAD ../domains/sykSH2SH2_pep_1A81_proteinonly.pdb GTSVSKGEED ../domains/mCherry_2H5Q_proteinonly.pdb GGSG ../domains/WW_pep_1JMQ_proteinonly.pdb:P SAWSHPQFEKGGGSGGGSGGGSWSHPQFEK --number 10 -d me0201_withpeptides.pdb

**Comment by guzmanfj** [comment]
Yes that's because that feature has still not been implemented :P 
The program is not doing anything with that argument yet

Will work on that soon!
